### ISSUE TITLE
Crash in pcube analysis with ixpeobssim 31.0.0

### ISSUE THREAD

**Issue by lucabaldini** [lucabaldini/ixpeobssim#725]
Dear Luca, eventually I was able to listen to our today's telecon. Inspired by weights discussion, I wanted to check the weighted analysis to compare with the unweighted with the following definition

```
kwargs_pcube = dict(algorithm='PCUBE', ebinalg='LIST', irfname='ixpe:obssim20240701_alpha075:v13', ebinning=E_BINNING, weights='True', acceptcorr='True', grayfilter='False')
```

and I got the following error:

```
File "/usr/local/share/IXPEobssim/ixpeobssim/irf/caldb.py", line 102, in irf_file_name
    raise RuntimeError('No simple weighting available for %s intent.', intent)
RuntimeError: ('No simple weighting available for %s intent.', 'obssim20240101_alpha075')
```

  Do you have any idea what I do wrong or what is wrong?

  Cheers,

  Michal.

**Comment by lucabaldini** [comment]
Ok, I found the offending bit, see
https://github.com/lucabaldini/ixpeobssim/pull/726/commits/9f9cd117cd59ef7ddac382e50ea4ef1a6f0c7b3e

It is getting more and more complicated to handle all the possible combinations of response files that we have to deal with.

**Comment by lucabaldini** [comment]
Pull request:
https://github.com/lucabaldini/ixpeobssim/pull/726/commits/9f9cd117cd59ef7ddac382e50ea4ef1a6f0c7b3e

**Comment by lucabaldini** [comment]
Shipped in 31.0.1
